oc_mirror.sh

```diff
@@ -78,33 +78,21 @@ function create_file_imageset() {
 
    imageset=$1
 
-   latest_release=$(oc-mirror list releases --channel candidate-${OPENSHIFT_RELEASE_STREAM}| tail -n1)
-
-   # Note that the archiveSize defines the maximum size, in GiB, of each file within the image set.
-   # This must be large enough to include all images in one file for the publish.
    cat > "${imageset}" << EOF
-apiVersion: mirror.openshift.io/v1alpha2
 kind: ImageSetConfiguration
-archiveSize: 16
-storageConfig:
-  local:
-    path: metadata
+apiVersion: mirror.openshift.io/v2alpha1
 mirror:
   platform:
-    architectures:
-      - "amd64"
-    channels:
-      - name: candidate-${OPENSHIFT_RELEASE_STREAM}
-        minVersion: $latest_release
-        maxVersion: $latest_release
-        type: ocp
+    graph: true
+    release: $OPENSHIFT_RELEASE_IMAGE
```

Comment:
I think a similar change need to be made for create_registry_imageset.

Reply:
As discussed, that's for mirror-to-mirror publish which is only valid for v1. I will remove it.